docs/CONFIG-VARS.md

```diff
@@ -185,7 +185,7 @@ Viya 4 supports 2 different types of certificate generators, cert-manager and op
 
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_TLS_GENERATOR | Which tool to use for certificate generation | string | openssl | false | Supported values: [`cert-manager`,`openssl`]. | viya, cluster-logging, cluster-monitoring |
+| V4_CFG_TLS_GENERATOR | Which tool to use for certificate generation | string | openssl | false | Supported values: [`cert-manager`,`openssl`]. If set to `cert-manager`, `cert-manager` will be installed during baselining. | viya, cluster-logging, cluster-monitoring |
```

Comment:
Yes, the others items in the `Tasks` column should go away. Only `baseline`

Reply:
@riragh I do agree that baseline should be added since `V4_CFG_TLS_GENERATOR` will now control the installation of cert-manager.

@thpang Does it make sense to remove the others because this flag also controls tasks within those roles as well? 
For `cluster-logging` & `cluster-monitoring` controls an env variable for the viya4-monitoring installation script and `viya` it dictates whether or not some transformers get added to the kustomization.yaml file when tls.yaml is run.

Reply:
Given the way the flag is used now sure. Why do the other items care what type of TLS generator is being used? Do they need to make specific calls to that generator? Again, more complicated than needed. We should install and register the TLS generator and then have other items that are interested read a configMap with the info or something. But it's good here.

Reply:
I can see that being a good way of doing it. A config map could be setup during the initial cert-manager installation and we have some steps in the logging/monitoring roles to check for that config map and take any values it needs.